web/src/emoji_picker.ts

```diff
@@ -169,17 +173,19 @@ export function rebuild_catalog(): void {
         catalog.set(category, emojis);
     }
 
-    const popular = [];
-    for (const codepoint of typeahead.popular_emojis) {
+    const frequently_used = [];
+    for (const codepoint of typeahead.frequently_used_emojis) {
         const name = emoji.get_emoji_name(codepoint);
         if (name !== undefined) {
             const emoji_dict = emoji.emojis_by_name.get(name);
             if (emoji_dict !== undefined) {
-                popular.push(emoji_dict);
+                frequently_used.push(emoji_dict);
             }
         }
     }
-    catalog.set("Popular", popular);
+    if (frequently_used.length > 0) {
```

Comment:
We always will have some emoji in "Frequently used" due to `popular_emojis`, right?